command.go

```diff
@@ -2161,3 +2161,116 @@ func (c *cmdsInfoCache) Get() (map[string]*CommandInfo, error) {
 	})
 	return c.cmds, err
 }
+
+//------------------------------------------------------------------------------
+
+type SlowLog struct {
+	ID        int64
+	CreatedAt time.Time
+	Costs     time.Duration
+	/*
+		ClientAddress,ClientName
+		There are also optional fields emitted only by Redis 4.0 or greater:
+		https://redis.io/commands/slowlog#output-format
+	*/
+	ClientAddress, ClientName string
+	Args                      []string
+}
+
+type SlowLogCmd struct {
+	baseCmd
+
+	val []SlowLog
+}
+
+var _ Cmder = (*SlowLogCmd)(nil)
+
+func NewSlowLogCmd(ctx context.Context, args ...interface{}) *SlowLogCmd {
+	return &SlowLogCmd{
+		baseCmd: baseCmd{
+			ctx:  ctx,
+			args: args,
+		},
+	}
+}
+
+func (cmd *SlowLogCmd) Val() []SlowLog {
+	return cmd.val
+}
+
+func (cmd *SlowLogCmd) Result() ([]SlowLog, error) {
+	return cmd.Val(), cmd.Err()
+}
+
+func (cmd *SlowLogCmd) String() string {
+	return cmdString(cmd, cmd.val)
+}
+
+func (cmd *SlowLogCmd) readReply(rd *proto.Reader) error {
+	_, cmd.err = rd.ReadArrayReply(func(rd *proto.Reader, n int64) (interface{}, error) {
+		cmd.val = make([]SlowLog, n)
+		for i := 0; i < len(cmd.val); i++ {
+			n, err := rd.ReadArrayLen()
+			if err != nil {
+				return nil, err
+			}
+			if n < 4 {
+				err := fmt.Errorf("redis: got %d elements in slowlog get, expected at least 4", n)
+				return nil, err
+			}
+
+			id, err := rd.ReadIntReply()
+			if err != nil {
+				return nil, err
+			}
+
+			createdAt, err := rd.ReadIntReply()
+			if err != nil {
+				return nil, err
+			}
+			createdAtTime := time.Unix(createdAt, 0)
+			costs, err := rd.ReadIntReply()
+			if err != nil {
+				return nil, err
+			}
+			costsDuration := time.Duration(costs)
```

Comment:
https://redis.io/commands/slowlog says

>The amount of time needed for its execution, in microseconds

Should it be `costs * time.Microsecond`?

Reply:
Yes, fix in https://github.com/go-redis/redis/pull/1368/commits/2d31f687b3738412951a1aa6958bd937c2365ff7#diff-9c42b524cb14ca001c61267826cbefb1R2236